ideascube/blog/templates/blog/content_form.html

```diff
@@ -24,7 +24,7 @@
                         {% endif %}
                     {% endfor %}
                 </div>
-                <div class="col pure-u-1 pure-u-md-2-3 layout-element">
+                <div class="col pure-u-1 pure-u-md-2-3 layout-element blog">
```

Comment:
`blog` doesn't feel like a great class name here.

In addition, this only changes it for the blog content in the edition form, but not when displaying the article?